docs/core/tutorials/using-on-macos.md

```diff
@@ -32,6 +32,16 @@ In this tutorial, you create three projects: a library project, tests for that l
 
 Start Visual Studio Code. Press <kbd>Ctrl</kbd>+<kbd>\`</kbd> (the backquote or backtick character) or select **View > Integrated Terminal** from the menu to open an embedded terminal in Visual Studio Code. You can still open an external shell with the Explorer **Open in Command Prompt** command (**Open in Terminal** on Mac or Linux) if you prefer to work outside of Visual Studio Code.
 
+### Launching Visual Studio Code from the command line
```

Comment:
You could add a numbered list with this steps:

1.  step x
2. step y

and so on.

Reply:
Great suggestion, I updated the PR.